instrumentation/akka/akka-http-10.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/akkahttp/server/AkkaHttpServerInstrumentationModule.java

```diff
@@ -29,9 +39,22 @@ public AkkaHttpServerInstrumentationModule() {
     super("akka-http", "akka-http-server");
   }
 
+  /*
+  Use a classLoaderMatcher to make this instrumentation only load when Akka HTTP is on the classpath.
+  This is because DispatchersInstrumentation is instrumenting an Actor class, which may be present when HTTP is not.
+  If another instrumentation uses Actor but not HTTP, this may cause Muzzle failures. Example: Play-WS
+   */
+  @Override
+  public ElementMatcher.Junction<ClassLoader> classLoaderMatcher() {
+    return hasClassesNamed("akka.http.scaladsl.HttpExt");
```

Comment:
If `akka.dispatch.Dispatchers` is in parent class loader and `akka-http` in child class loader then dispatchers won't get instrumented.

Reply:
I hadn't considered multiple class loaders. I'll rework this to use your original suggestion of a separate InstrumentationModule.